resources/assets/js/helpers/forms.ts

```diff
@@ -0,0 +1,47 @@
+import { FormikProps, FormikValues } from "formik";
+import _ from "lodash";
+import { MutableRefObject } from "react";
+
+export const focusOnElement = (id: string): void => {
+  const element = document.getElementById(id);
+  if (element) {
+    element.focus();
+  }
+};
+
+export const validateAllForms = async (
+  refs: MutableRefObject<MutableRefObject<FormikProps<FormikValues>>[]>,
```

Comment:
I realized while poking around that in this function and submitAllForms, we just need the inner array of refs, there's no need to pass in the array wrapped in another MutableRefObject.

Reply:
Noted. Thanks again for the help. I'm going to keep looking for a solution to the `focusOnElement` issue tomorrow.